docker/nginx.conf

```diff
@@ -0,0 +1,20 @@
+worker_processes 1;
+
+events { worker_connections 1024; }
+
+http {
+  include /etc/nginx/mime.types;
+
+  server {
+  	listen 80;
```

Comment:
Maybe good to expose this at a port `> 1024`, because otherwise this only works for root users and if you don't already have a webserver running.